helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ResourceMonitor.java

```diff
@@ -335,12 +340,21 @@ private void resetResourceStateGauges() {
     _numOfPartitionsInExternalView.updateValue(0L);
     _numLessMinActiveReplicaPartitions.updateValue(0L);
     _numLessReplicaPartitions.updateValue(0L);
+    _oneOrManyPartitionsMissingTopStateRealTimeGuage.updateValue(0L);
```

Comment:
This is very long name. Also it already represents how many partitions missing as aggregation. Better be: _numPartitionsMissingTopStateRealTime

Reply:
TFTR @junkaixue! I agree with long name but i just wanted it to be verbose. Actually counter value doesn't indicate number of partitions missing top state (not even at aggregated level) because this number is continuously increasing once started. Any positive value of this counter just indicates that there could be at least one partition (could be more) with missing top states.